crates/ra_assists/src/handlers/add_function.rs

```diff
@@ -168,7 +176,7 @@ fn fn_args(
             None => String::from("arg"),
         };
         arg_names.push(arg_name);
-        arg_types.push(match fn_arg_type(ctx, &arg) {
+        arg_types.push(match fn_arg_type(ctx, target_module, &arg) {
```

Comment:
This means that if the path to a type can't be resolved, the assist won't be offered. Now that I'm writing this, that probably deserves a test :)

Reply:
Trying to test this, I noticed that I fall back to `()` when the type of an argument couldn't be resolved.
What do you think, is it better to offer to generate a function with wrong arguments or to only ever generate functions if the signature is gonna be correct?